File: Android/ConsultaCEP/app/src/main/java/com/leocoesta/consultacep/MainActivity.java
package com.leocoesta.consultacep;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import java.util.concurrent.ExecutionException;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final EditText cepET = findViewById(R.id.etMain_cep);
        final TextView respostaTV = findViewById(R.id.etMain_resposta);

        Button btnBuscarCep = findViewById(R.id.btnMain_buscarCep);

        btnBuscarCep.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                try {
                    CEP retornoConsulta = new HttpService(cepET.getText().toString()).execute().get();
                    respostaTV.setText(retornoConsulta.toString());
                } catch (InterruptedException e) {
                    e.printStackTrace();
                } catch (ExecutionException e) {
                    e.printStackTrace();
                }
            }
        });
    }
}

File: Android/TesteMascara/app/src/main/java/com/leocoesta/testemascara/MainActivity.kt
package com.leocoesta.testemascara

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import br.com.sapereaude.maskedEditText.MaskedEditText

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val edit = findViewById<MaskedEditText>(R.id.teste)
        val bt = findViewById<Button>(R.id.button)

        bt.setOnClickListener { println(edit.rawText) }

    }
}
